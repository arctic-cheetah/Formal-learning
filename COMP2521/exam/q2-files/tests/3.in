50 70 30 20 44 80 90 92 94 96 

