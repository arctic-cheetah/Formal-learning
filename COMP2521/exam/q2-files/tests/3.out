
Displaying tree (sideways) 
						96
					94
				92
			90
		80
	70
50
		44
	30
		20

 -------  
nodesNotBalanced(t) returns: 5 
 -------  
